### PR TITLE
cap action packets at 30 yalms during combat

### PR DESCRIPTION
**Pull request by takhlaq**

### CODE REVIEW

**Comment by teschnei** [comment]
Hmm, is it actually retail, though?

If it's for the "alliance lag", I don't think capping action packets at 30 really makes much of a difference.  How often do you have tons of people farther than 30 yalms but closer than 50?

**Comment by R3P0FFXI** [comment]
@teschnei @takhlaq  the issue isnt being reported by players across the zone its being reported by players in a small BCNM area or in a party or in a big alliance fight. In retail i think it is 50 yalms. It seems to be something related directly to players in the party/alliance.

**Comment by takhlaq** [comment]
>Hmm, is it actually retail, though?

ya, check your backlog in windower

File: src/map/zone_entities.cpp [hunk 1]
```diff
@@ -819,12 +819,16 @@ void CZoneEntities::PushPacket(CBaseEntity* PEntity, GLOBAL_MESSAGE_TYPE message
             }
             case CHAR_INRANGE:
             {
+                // todo: rewrite packet handlers and use enums instead of rawdog packet ids
+                // 30 yalms if action packet and in combat, 50 otherwise
+                const int checkDistanceSq = packet->id() == 0x0028 && PEntity->PAI->IsEngaged() ? 900 : 2500;
```
Comment (teschnei) [inline src/map/zone_entities.cpp]: why the engaged check?  i don't think that's part of anything that was said in windower

Comment (takhlaq) [reply src/map/zone_entities.cpp]: i made poopy

**Pull request merged**